conservator-framework/src/ConservatorFrameworkFactory.cpp

```diff
@@ -4,19 +4,19 @@
 
 #include "ConservatorFrameworkFactory.h"
 
-ConservatorFramework ConservatorFrameworkFactory::newClient(string connectString) {
-    return ConservatorFramework(connectString);
+unique_ptr<ConservatorFramework> ConservatorFrameworkFactory::newClient(string connectString) {
+    return unique_ptr<ConservatorFramework>(new ConservatorFramework(connectString));
```

Comment:
I think `make_unique` is preferred to a direct allocation: http://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique